src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -114,7 +114,7 @@
 
       <AspNetCore31RuntimePackRids Include="@(AspNetCore30RuntimePackRids)" />
       <AspNetCore50RuntimePackRids Include="@(AspNetCore31RuntimePackRids);linux-musl-arm;win-arm64" />
-      <AspNetCoreRuntimePackRids Include="@(AspNetCore50RuntimePackRids);osx-arm64" />
```

Comment:
We want to keep this how it is and not accept this change, since 6.0 will support osx-arm64 but 5.0 does not.